### DISCUSSION TITLE
Could not find any suitable route

### DISCUSSION THREAD

**Discussion by HotCrypt** [lightningnetwork/lnd#8319] [Q&A]
HI, I'm trying to balance a channel which I opened with 1ml, I want to send 123ksat to remote, below the info:

```
                                   Commit ------- Balance --------- --- Fee ----
Alias or Pubkey           Capacity   Fee        Local       Remote   Base PerMil
-------------------- ------------- ------ ------------ ------------ ----- ------
1ML.com node ALPHA          271745  25591       245824            0     0      0
```

```
`rebalance.py --list-candidates --show-only 897377410125987841`

Channel ID:       897377410125987841
Alias:            1ML.com node ALPHA
Pubkey:           0217890e3aad8d35bc054f43acc00084b25229ecff0ab68debd82883ad65ee8266
Channel Point:    5abbb8dfbecffdc45a845bd8517cce34bd40f2e8d585144f9c7778b4b7973891:1
Local ratio:      1.000
Fee rates:        0ppm (own), 1ppm (peer)
Capacity:            271,745
Remote available:          0
Local available:     246,566
Rebalance amount:   -123,283

```

I tried this command to balance but I get this:

```
`~ $ rebalance.py --amount 123283 --from 897377410125987841 --to 897377410125987841 `

> Sending 123,283 satoshis from channel 897377410125987841 with 1ML.com node ALPHA back through 1ML.com node ALPHA.
> Setting fee limit to 1,000 mSAT (8ppm).
> Could not find any suitable route

```

Logs Lnd: 

> ERR] RPCS: [/lnrpc.Lightning/QueryRoutes]: unable to find a path to destination
> Dec 28 22:37:24 raspberrypi lnd[225492]: 2023-12-28 22:37:24.232 
> [INF] IRPC: Canceled invoice 093603fc2e91bde82556e42a40d4de62ec494dda4f1d59ce58a664971e2def3c

Thank you.

**Comment by microlancer** [comment]
It's possible that there is no route available at that fee rate, and also possible that there is no route available at any fee rate. You might need to experiment with different sizes and fees to find a suitable route.

**Comment by HotCrypt** [comment]
Thx I'm going to try again.

The command fails instantly, it does not try any hops.
